react/src/base/inputs/Date.stories.js

```diff
@@ -31,55 +31,103 @@ which you may need to remove before submitting the form.
 };
 
 export const dateInput = () => (
-  <SprkTextInput
-    label="Date"
-    name="date"
-    placeholder="01/01/2019"
-  />
+  <SprkInputContainer>
+    <SprkLabel htmlFor="date-1">Date</SprkLabel>
+    <SprkInput id="date-1" placeholder="01/01/2021" />
+  </SprkInputContainer>
 );
 
 dateInput.story = {
   name: 'Default',
   parameters: {
-    jest: [
-      'SprkTextInput',
-    ]
+    jest: ['SprkInput', 'SprkLabel', 'SprkInputContainer'],
   },
 };
 
 export const invalidDateInput = () => (
-  <SprkTextInput
-    label="Date"
-    name="date"
-    placeholder="01/01/2019"
-    valid={false}
-    errorMessage="There is an error on this field."
-  />
+  <SprkInputContainer>
+    <SprkLabel htmlFor="date-2">Date</SprkLabel>
+    <SprkInput
+      id="date-2"
+      placeholder="01/01/2021"
+      isValid={false}
+      ariaDescribedBy="invalid-error"
+    />
+    <SprkFieldError id="invalid-error">
+      <SprkIcon
+        iconName="exclamation-filled"
+        additionalClasses="sprk-b-ErrorIcon"
+        aria-hidden="true"
+      />
+      <div className="sprk-b-ErrorText">There is an error on this field.</div>
+    </SprkFieldError>
+  </SprkInputContainer>
 );
 
 invalidDateInput.story = {
   name: 'Invalid',
   parameters: {
     jest: [
-      'SprkTextInput',
-    ]
+      'SprkInput',
+      'SprkLabel',
+      'SprkInputContainer',
+      'SprkFieldError',
+      'SprkIcon',
+    ],
   },
 };
 
 export const disabledDateInput = () => (
+  <SprkInputContainer>
+    <SprkLabel htmlFor="date-3" isDisabled>
+      Date
+    </SprkLabel>
+    <SprkInput id="date-3" placeholder="01/01/2021" isDisabled />
+  </SprkInputContainer>
+);
+
+disabledDateInput.story = {
+  name: 'Disabled',
+  parameters: {
+    jest: ['SprkInput', 'SprkLabel', 'SprkInputContainer'],
+  },
+};
+
+export const legacy = () => (
+  <SprkTextInput label="Date" name="date" placeholder="01/01/2021" />
+);
+
+legacy.story = {
+  name: 'Legacy (Deprecated)',
+  parameters: {
+    jest: ['SprkTextInput'],
+  },
+};
+
+export const legacyInvalidDateInput = () => (
   <SprkTextInput
     label="Date"
     name="date"
-    placeholder="01/01/2019"
-    disabled
+    placeholder="01/01/2021"
+    valid={false}
```

Comment:
We'll need to double check this.  Looks like we added  `valid={false}?`

Reply:
It's just that everything got shifted down for the legacy stories. This is the invalid story and should have valid={false}